pkg/daemon/update.go

```diff
@@ -541,17 +541,13 @@ func (dn *Daemon) writeUnits(units []ignv2_2types.Unit) error {
 			glog.V(2).Infof("Wrote systemd unit dropin at %s", dpath)
 		}
 
-		if u.Contents == "" {
-			continue
-		}
-
-		glog.Infof("Writing systemd unit %q", u.Name)
-
 		fpath := filepath.Join(pathSystemd, u.Name)
 
 		// check if the unit is masked. if it is, we write a symlink to
 		// /dev/null and continue
 		if u.Mask {
+			glog.Infof("Writing systemd unit %q", u.Name)
```

Comment:
If you want a message here, shouldn't this message be something like "Symlinking.."

Reply:
Not something I wanted really; just left the logic as it was before (It should print exactly what it was printing before).

Reply:
That logging line was moved into the `if u.Mask` block by this PR. Previously it was on line 548 before the `if u.Mask.` It's placement at line 549 is inaccurate and it now appears twice in the function. The same logging on line 566 is correct. You can either change the logging on line 549 as suggested above or remove it entirely.

Reply:
I'll remove it.

Reply:
Done

Reply:
Awesome, thanks!